src/main/java/com/siemens/ct/exi/main/cmd/EXIficientCMD.java

```diff
@@ -102,6 +102,8 @@ public class EXIficientCMD {
 	public static final String XSD_SCHEMA = "-" + SchemaOption.xsdSchema;
 	public static final String SCHEMA = "-" + SchemaOption.schema;
 
+	public static final String SCHEMA_ID = "-schemaId";
+	public static final String BYTE_ID = "-byteId";

```

Comment:
This option does not exit in the [EXI](https://www.w3.org/TR/exi/) specification?